scripts/release/util.js

```diff
@@ -49,12 +49,36 @@ const getElementList = async (directory) => {
     });
 };
 
+/*
+ * Return remove and replace unwanted text for md.
+ */
+const sanitize = (text) => {
+  return text?.replace('\n', ' ').replaceAll('\r', ' ').replaceAll('`', "'");
+};
+
+/*
+ * List for generate document from TS.
+ * Each item contains require fields.
+ * entry - path of ts file that uses to generate doc.
+ * tsconfig - tsconfig path.
+ * title - title for header level 1.
+ */
+const generateDocList = [
+  {
+    entry: '../packages/elements/src/tree/managers/tree-manager.ts',
+    tsconfig: '../packages/elements/tsconfig.json',
+    title: 'Tree Manager'
+  }
+];
+
 export {
+  sanitize,
```

Comment:
sanitize is used in `utils.injector.js` only. Why it's defined in here as util?

Reply:
Agreed that it should placed at utils.injector.js.